extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -60,23 +60,47 @@ function createDicomWebApi(dicomWebConfig, userAuthenticationService) {
     supportsReject,
     staticWado,
     singlepart,
+    omitQuotationForMultipartRequest,
+    acceptHeader
   } = dicomWebConfig;
 
   const dicomWebConfigCopy = JSON.parse(JSON.stringify(dicomWebConfig));
 
+  //If accept header is not specified, set as multipart
+  let formattedAcceptHeader = acceptHeader ?? []
```

Comment:
You are dealing with slightly different lists - the formatted accept header is a single line accept header that has parts separated by semi-colon, while the acceptHeader is a list of header values that are complete - for example:
acceptHeader: [ 'multipart/related; type=image/jpls; q=1', 'multipart/related; type=application/octet-stream; q=0.1' ]
so that one can prioritize the headers.

Reply:
it will mean I have to split each header by the ; separator to add the quotation and reserialize it ? 
At it going to provided by the user as an array, the user can already decide priority when building the array that we are going to use "as it"? can you detail the usecase for quotation ? 

In my testing my header made in this file doesn't change anything in dicom web requests, do you have an idea why my changes are not propagated in the requests ?